### ISSUE TITLE
context in template isn't passed

### ISSUE THREAD

**Issue by francescoagati** [monkberry/monkberry#23]
The context isn't passed when printed on template.
Here an example http://jsfiddle.net/z8pzsdyv/11/

**Comment by elfet** [comment]
Yes, this is true. In render state there is no context. Context will appear after. 
I know how to fix it, need to move all static constructions with `this` or `this.context` into spots. 
Here is working example: http://jsfiddle.net/z8pzsdyv/12/
But note, what there is no dynamic bindings with context in template: https://github.com/monkberry/monkberry/blob/master/monkberry.js#L75

**Comment by francescoagati** [comment]
yes is nice for using some context also in printing variables

**Comment by francescoagati** [comment]
but i get error also with filters at the first rendering if i call a filter

**Comment by elfet** [comment]
> but i get error also with filters at the first rendering if i call a filter

True. You can add a variable to call.

That purpose of doing stuff like this?

**Comment by francescoagati** [comment]
i wanna pass context and filters to template. but whe the template is rendered i get an error
